nodes/nomos-node/node/src/main.rs

```diff
@@ -111,7 +74,41 @@ async fn main() -> Result<()> {
         None,
     )
     .map_err(|e| eyre!("Error encountered: {}", e))?;
-    let _ = app.handle().start_all_services().await;
+
+    let services_to_start = get_services_to_start(
+        &app,
+        must_blend_service_group_start,
+        must_da_service_group_start,
+    )
+    .await?;
+
+    let _ = app.handle().start_service_sequence(services_to_start).await;
+
     app.wait_finished().await;
     Ok(())
 }
+
+async fn get_services_to_start(
+    app: &Overwatch<RuntimeServiceId>,
+    must_blend_service_group_start: bool,
+    must_da_service_group_start: bool,
+) -> Result<Vec<RuntimeServiceId>, OverwatchError> {
+    let mut service_ids = app.handle().retrieve_service_ids().await?;
```

Comment:
I'm thinking, would it make sense for a service to declare the service group they belong to, maybe in their `ServiceData` implementation? Something like `const SERVICE_GROUP: Option<&'static str>`? I am wondering whether the concept of a service belonging to a group should be in the node or in the service itself. What do you think? Since services depend on each other, I think there is an acyclic path between them, so the service that does not depend on anything might want to export its service group, and other services might simply refer to that to avoid one service changing its service group while the others don't. Hope I made it clear enough what I am trying to say...